charts/tidb-cluster/templates/fullbackup-cronjob.yaml

```diff
@@ -43,28 +43,50 @@ spec:
               mkdir -p /data/${dirname}/
               cp /savepoint-dir/savepoint /data/${dirname}/
 
-              /mydumper \
+              /usr/bin/mydumper \
```

Comment:
Mydumper and uploader are already in `$PATH`, so it's unnecessary to specify the full path.